src/structures/Message.js

```diff
@@ -64,7 +64,7 @@ class Message extends Base {
          * ID for the Chat that this message was sent to, except if the message was sent by the current user.
          * @type {string}
          */
-        this.from = typeof (data.from) === 'object' ? data.from._serialized : data.from;
+        this.from = (typeof (data.from) === 'object' && data.author !== null) ? data.from._serialized : data.from;
```

Comment:
```suggestion
        this.from = (typeof (data.from) === 'object' && data.from !== null) ? data.from._serialized : data.from;
```

Reply:
Yes, sorry, my bad. Good old copy and paste error. Should I change it and resubmit a new commit?

Reply:
No worries. You should be able to just make the changes and push a new commit to your branch and it should show up here